source/blog/2016-03-11-docker-credentials-store.html.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Docker credentials store
+author: runcom
+date: 2016-03-15 10:00:00 UTC
+tags: docker
+published: true
+comments: true
+---
+
+One security feature in the upcoming Docker 1.11 is the ability to use an external credentials store for registry authentication. The new version will automatically detect a configured external store, if it is available, and use it instead of the JSON file. We'll be talking more about this in a few paragraphs, but first, let's see how Docker is currently storing credentials.
+
+Login and logout
+=
+
+Today, Docker stores the credentials used for registry authentication inside a JSON file (usually in `$HOME/.docker/config.json`).
+Its format is pretty simple:
+
+```bash
```

Comment:
these `bash` tags aren't interpreted and theyt are shown in the online article: http://www.projectatomic.io/blog/2016/03/docker-credentials-store/